deploy-argocd-apps.sh

```diff
@@ -1,19 +1,18 @@
 #!/bin/bash -e
 set -ex
-# Deploy Applications to ArgoCD on EKS Cluster
 
-if [ "$#" -ne 2 ] || ([ "$2" != "production" ] && [ "$2" != "staging" ] ); then
-  echo "Usage: $0 <ArgoCD server> <staging/production>" >&2
+# Deploy Applications to ArgoCD on EKS Cluster
```

Comment:
I think we can drop the comment. It mentions EKS while EKS is not the only case for it, and its location is not relevant to the comment, as far as I see it.